qa/suites/fs/upgrade/featureful_client/old_client/tasks/2-upgrade.yaml

```diff
@@ -5,8 +5,23 @@ tasks:
     mon.a:
     mon.b:
 - print: "**** done install.upgrade both hosts"
-- ceph.stop: [mds.*]
-- ceph.restart:
-    daemons: [mon.*, mgr.*, osd.*, mds.*]
-    mon-health-to-clog: false
+upgrade-sequence:
+   sequential:
+   - ceph.restart:
+       daemons: [mon.*, mgr.*]
+       mon-health-to-clog: false
+       wait-for-healthy: false
+   - exec:
+       mon.a:
+         - ceph config set global mon_warn_on_msgr2_not_enabled false
```

Comment:
probably need to whitelist the warning too, since it'll possibly pop up between this mons restarting and then getting disabled.

Reply:
Thanks! I'll add that.